sklearn/multiclass.py

```diff
@@ -574,6 +576,8 @@ def predict(self, X):
             Predicted multi-class targets.
         """
         Y = self.decision_function(X)
+        if self.n_classes_ == 2:
+            return self.classes_[(Y > 0).astype(np.int)]
```

Comment:
no we should assume that base_estimator actually works. Otherwise everything in sklearn is broken ;)